Assets/Mirror/Runtime/NetworkConnection.cs

```diff
@@ -216,19 +216,7 @@ public virtual void TransportReceive(byte[] buffer)
             NetworkReader reader = new NetworkReader(buffer);
             if (Protocol.UnpackMessage(reader, out ushort msgType))
             {
-                if (logNetworkMessages)
-                {
-                    if (Enum.IsDefined(typeof(MsgType), msgType))
-                    {
-                        // one of Mirror mesage types,  display the message name
-                        Debug.Log("ConnectionRecv con:" + connectionId + " msgType:" + (MsgType)msgType + " buffer:" + BitConverter.ToString(buffer));
-                    }
-                    else
-                    {
-                        // user defined message,  display the number
-                        Debug.Log("ConnectionRecv con:" + connectionId + " msgType:" + msgType + " buffer:" + BitConverter.ToString(buffer));
-                    }
-                }
+                if (logNetworkMessages) { Debug.Log("ConnectionRecv con:" + connectionId + " msgType:" + msgType + " buffer:" + BitConverter.ToString(buffer)); }
```

Comment:
Why?  I added that so that you could see what message we were sending instead of just a number